x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { i18n } from '@kbn/i18n';
+import { EuiTabbedContent, EuiSpacer, EuiTabbedContentTab } from '@elastic/eui';
+
+import { usePolicyDetailsSelector } from '../policy_hooks';
+import {
+  isOnPolicyFormPage,
+  isOnPolicyTrustedAppsPage,
+} from '../../store/policy_details/selectors';
+
+import { PolicyTrustedAppsLayout } from '../trusted_apps/layout';
+import { PolicyFormLayout } from '../policy_forms/components';
+
+export const PolicyTabs = React.memo(() => {
+  const history = useHistory();
+  const isInSettingsTab = usePolicyDetailsSelector(isOnPolicyFormPage);
+  const isInTrustedAppsTab = usePolicyDetailsSelector(isOnPolicyTrustedAppsPage);
+
+  const tabs = [
+    {
+      id: 'settings',
+      name: i18n.translate('xpack.securitySolution.endpoint.policy.details.tabs.policyForm', {
+        defaultMessage: 'Policy settings',
+      }),
+      content: (
+        <>
+          <EuiSpacer />
+          <PolicyFormLayout />
+        </>
+      ),
+    },
+    {
+      id: 'trustedApps',
+      name: i18n.translate('xpack.securitySolution.endpoint.policy.details.tabs.trustedApps', {
+        defaultMessage: 'Trusted applications',
+      }),
+      content: (
+        <>
+          <EuiSpacer />
+          <PolicyTrustedAppsLayout />
+        </>
+      ),
+    },
+  ];
+
+  const getInitialSelectedTab = () => {
+    let initialTab = tabs[0];
+
+    if (isInSettingsTab) initialTab = tabs[0];
+    else if (isInTrustedAppsTab) initialTab = tabs[1];
+    else initialTab = tabs[0];
+
+    return initialTab;
+  };
+
+  const onTabClickHandler = useCallback(
+    (selectedTab: EuiTabbedContentTab) => {
+      history.push(history.location.pathname.replace(/\/([^\/]*)$/, `/${selectedTab.id}`));
```

Comment:
Again, don't really like this kind of replacements. Suggestions?

Reply:
You should use `getPolicyDetailPath()` to get the path instead of manipulating the current one and doing a replace on it.

Reply:
I don't think this is going to solve this. I need to replace the last `/something` by `/settings` or `/trustedApps`

Reply:
ok. I just noticed that `getPolicyDetailsPath` is unchanged. my suggestion is to add a param to that function that allows us to define the tab. Param should be optional and default to the Settings route, so that existing usages of it remain un-impacted. something like:

```typescript
export const getPolicyDetailPath = (policyId: string, search?: string, tab?: 'settings' | 'artifacts') => {}
```